### PR TITLE
users/config, users/faq: Document low priority setting

### PR DESCRIPTION
**Pull request by calmh**
https://github.com/syncthing/syncthing/pull/4675

### CODE REVIEW

File: users/config.rst [hunk 1]
```diff
@@ -124,6 +124,7 @@ The following shows an example of the default configuration file (IDs will diffe
             <kcpReceiveWindowSize>128</kcpReceiveWindowSize>
             <defaultFolderPath>~</defaultFolderPath>
             <minHomeDiskFreePct>0</minHomeDiskFreePct>
+            <setLowPrio>true</setLowPrio>
```
Comment (AudriusButkevicius) [inline users/config.rst]: So is it prio or priority? It should be priority everywhere I think.

Comment (calmh) [reply users/config.rst]: Whoops

**Comment by AudriusButkevicius** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as d0c292e02767b6c8dacf03580f5fa6a7dec9fe9c. Thanks, @calmh!

**Pull request closed without merging**